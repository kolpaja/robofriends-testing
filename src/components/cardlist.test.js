import { shallow } from 'enzyme';
import React from "react"
import CardList from "./CardList"
import { shallowToJson } from 'enzyme-to-json';

it("CardList comp test", () => {

    const mockRobots = [
        {
            id: 1,
            name: "test mock",
            username: "testmock1",
            email: "test@gmail.com",
        }
    ]
    const card = shallow(<CardList robots={mockRobots} />)
    expect(shallowToJson(card)).toMatchSnapshot();
})