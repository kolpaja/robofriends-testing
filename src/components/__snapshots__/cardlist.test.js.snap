// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList comp test 1`] = `
<div>
  <Card
    email="test@gmail.com"
    id={1}
    key="0"
    name="test mock"
  />
</div>
`;
